fetch/main.js

```diff
@@ -1,17 +1,20 @@
 async function getData() {
-  try {
     const response = await fetch('https://jsonplaceholder.typicode.com/users');
+if (!response.ok){
+  new Error(`The reponse was unsucessful with the status Code - ${reponse.status}`)
+}
+  try {
```

Comment:
This was correct before. The `try` needs to go around the `await fetch` call as well or you won't get any exceptions thrown by fetch.